releasenotes/notes/observable-eval-primitives-e1fd989e15c7760c.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Added `~qiskit.algorithms.observables_evaluator` with
+    :class:`~qiskit.primitives.BaseEstimator` as ``init`` parameter. It will soon replace
+    `~qiskit.algorithms.aux_ops_evaluator`.
+deprecations:
+  - |
+    Using `~qiskit.algorithms.aux_ops_evaluator` will now issue a ``PendingDeprecationWarning``.
```

Comment:
This should stage it will be deprecated in  a future release and removed etc. Also it should note the replacement. These sections will be separated in the release notes so they need to have enough context/information to standalone

Reply:
Ok, did not know about the separation! Thanks.